### PR TITLE
N30: godot-rust

### PR DESCRIPTION
**Pull request by Bromeon**
_Part of #912_

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/030/index.md [hunk 1]
```diff
@@ -76,6 +76,35 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [godot-rust](https://github.com/godot-rust/godot-rust)
```
Comment (ozkriff) [inline content/news/030/index.md:79]:
```suggestion
### [godot-rust][gd-github]
```

**Review by ozkriff — APPROVED** [review]

File: content/news/030/index.md [hunk 2]
```diff
@@ -76,6 +76,35 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [godot-rust][gd-github]
+
+godot-rust ([GitHub][gd-github], [Discord][gd-discord], [Twitter][gd-twitter])
```
Comment (ozkriff) [inline content/news/030/index.md:81]: maybe add some image or at lesr the project's logo?

File: content/news/030/index.md (hunk 1 above)
Comment (Bromeon) [reply content/news/030/index.md]: Good catch, changed!

File: content/news/030/index.md (hunk 2 above)
Comment (Bromeon) [reply content/news/030/index.md:81]: Would it be OK to showcase some games, like in https://twitter.com/GodotRust/status/1482754475483926531?
Or should I limit it to the godot-rust logo?

Comment (ozkriff) [reply content/news/030/index.md:81]: i think it should be fine (we're already doing a [similar thing](https://gamedev.rs/news/003/#gfx-rs-v0-4) from time to time anyway)

**Comment by Bromeon** [comment]
Added showcase image (44.4 kB) and moved description to _Engine Updates_ category, as discussed on Discord.

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**